app/services/database.py

```diff
@@ -471,7 +473,6 @@ def update_token(self, session,record_id, user_name):
 
     def get_connection_string(self, server, database, client_id):
         try:
-            dns = f'mssql+pyodbc://FLM-VM-COGAIDEV/AudioTrans?driver=ODBC+Driver+17+for+SQL+Server'
```

Comment:
dns = f'mssql+pyodbc://FLM-VM-COGAIDEV/AudioTrans?driver=ODBC+Driver+17+for+SQL+Server'

Can you make it fully configurable?

Reply:
ODBC driver now is configurable on this changeset e1ae681